src/or/connection.c

```diff
@@ -1146,7 +1150,8 @@ static connection_t *
 connection_listener_new(const struct sockaddr *listensockaddr,
```

Comment:
If we're adding new arguments here, we need to document them.

Reply:
Fixed in 714ebd1e2b1878d17cc703f19f62c39c206b9473.